packages/anzen/README.md

```diff
@@ -391,16 +415,56 @@ For failure cases:
 ```js
 import { Result } from '@daisugi/anzen';
 
-const error = await Result.promiseAll([
-  async () => Result.failure('foo')
-]).getError();
+const promise = async () => Result.failure('foo');
+const error = (await Result.promiseAll([
+  promise(),
+])).getError();
 // 'foo'
 ```
 
 [:top: Back to top](#-table-of-contents)
 
 ---
 
+### `#unwrapPromiseAll(defaultValue, whenResults)`
+
+Similar to `promiseAll`, but unwraps the value from the Result.
+
+#### Example:
+
+```js
+import { Result } from '@daisugi/anzen';
+
+const promise = async () => Result.success('foo');
```

Comment:
Having two different values for _default_ and result might help clarify the code intent:

```suggestion
const promise = async () => Result.success('bar');
```